cmd/storage-rest-client.go

```diff
@@ -218,6 +220,8 @@ func (client *storageRESTClient) AppendFile(volume, path string, buffer []byte)
 	values.Set(storageRESTVolume, volume)
 	values.Set(storageRESTFilePath, path)
 	reader := bytes.NewBuffer(buffer)
+	globalNetStats.Transmit(uint64(len(buffer)))
+	defer globalNetStats.FinishTransmitting(uint64(len(buffer)))
 	respBody, err := client.call(storageRESTMethodAppendFile, values, reader, -1)
```

Comment:
Transfers happens on this line - what if there's an error? Looks like the count will go wrong here?

Reply:
moved the counting after transfer success